pkgs/by-name/ba/bandcamp-collection-downloader/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  jre,
+  gradle_8,
+
+  makeWrapper,
+  stdenv,
+  fetchFromGitLab,
+}:
+let
+  gradle = gradle_8;
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "bandcamp-collection-downloader";
+  version = "0-unstable-2024-10-29";
+
+  nativeBuildInputs = [
+    gradle
+    makeWrapper
+  ];
```

Comment:
can you move `nativeBuildInputs` to be below `src`/`mitmCache`? generally `src` is the first thing after pname/version. [nixpkgs-hammering](https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/attribute-ordering.md) documents the complete ordering but the remaining attributes mostly aren't so strict.